plugins/json_visualization_templates/json_visualization.html

```diff
@@ -0,0 +1,69 @@
+{% import 'forms.html' as forms %}
+
+<html>
+    <head>
+        <link rel="stylesheet" type="text/css" href="{{url_for('static', filename='forms.css')}}" media="screen" />
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/styles/rainbow.min.css">
+    </head>
+
+    <body>
+        <div class="qhana-microfrontend">
+            <details id="visualization-options" open>
+                <summary>Visualization Options</summary>
+                {% call forms.render_form(target="microfrontend") %}
+                    {{ forms.render_fields(schema, values=values, errors=errors) }}
+                    <div class="qhana-form-buttons">
+                    {{ forms.submit("validate", target="microfrontend")}}
+                    {{ forms.submit("submit", action=process)}}
+                    </div>
+                {% endcall %}
+            </details>
+        </div>
+
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.5.1/highlight.min.js"></script>
+
+        <pre>
+            <code id="json-preview" class="qhana-data-preview" style="flex-direction: row; flex-wrap: wrap;
+            max-height: 750px; overflow: auto; background: transparent" hidden>
+            </code>
+        </pre>
+
+        <script>
+        function httpGet(theUrl) {
+            let xmlHttpReq = new XMLHttpRequest();
```

Comment:
You should look up the "new" fetch api. While it is still not supported by every browser it is old enough to be used. (this does not have to be changed)